compiler-rt/lib/tsan/rtl/tsan_interface.h

```diff
@@ -219,14 +219,14 @@ __extension__ typedef __int128 a128;
 
 // Part of ABI, do not change.
```

Comment:
Is it still following the ABI - if we do not static_cast it to int, then I don't think this is strictly required to follow ABI.

Reply:
Does #114916 help?